bin/lowly.js

```diff
@@ -44,4 +48,46 @@ input.forEach(file => {
     }).catch(err => {
       console.error(err)
     })
+}
+
+input.forEach(ip => {
```

Comment:
Could you remove all this fancy check to see if the files are directories, and just check if the string receveid by the `forEach` is an supported image? The module [resize-img](https://github.com/kevva/resize-img) supports `bmp`, `jpg` and `png`.